### PR TITLE
Adds more informations about our code of conduct

### PR DESCRIPTION
**Pull request by mlhamel**
Just had the idea that should probably advertise more about our code of conduct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +22,7 @@ Build your dev docker image:
 
 Run initial DB setup:
 
-    $ docker-compose run web python manage.py syncdb --migrate
+    $ docker-compose run web python manage.py migrate
```
Comment (jordigh) [inline README.md]: Unrelated change?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 Deployments are triggered automatically when pushing to the *master* branch.
 
+## [Code of Conduct](https://github.com/mtlpy/code-of-conduct)
+
+Montréal-Python has adopted a Code of Conduct that we expect project participants to adhere to. Please read [the full text](https://github.com/mtlpy/code-of-conduct) so that you can understand what actions will and will not be tolerated.
```
Comment (peristeri) [inline README.md]: > Please read the full review for overview of the type of behaviours deemed inappropriate. 

Here's something a bit more formal.

File: README.md (hunk 1 above)
Comment (mlhamel) [inline README.md]: oups you are totally right. i had some local changes. will fix that

File: README.md (hunk 2 above)
Comment (mlhamel) [inline README.md]: Maybe something like:

```
If you need more informations please read the full review for an overview of the types of behaviours deemed inappropriate.

```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,10 @@
 
 Deployments are triggered automatically when pushing to the *master* branch.
 
+## [Code of Conduct](https://github.com/mtlpy/code-of-conduct)
+
+Montréal-Python has adopted a Code of Conduct that we expect project participants to adhere to. If you need more informations please read [the full text](https://github.com/mtlpy/code-of-conduct) for an overview of the types of behaviours deemed inappropriate. Please read so that you can understand what actions will and will not be tolerated.
```
Comment (merwok) [inline README.md]: information* (it’s uncountable).

Maybe break lines at a reasonable column (72 or 99)?
